gnome-initial-setup/pages/location/gis-location-page.c

```diff
@@ -812,6 +812,9 @@ gis_location_page_init (GisLocationPage *page)
 void
 gis_prepare_location_page (GisDriver *driver)
 {
+  if (gis_driver_is_live_session (driver))
+    return;
```

Comment:
The commit message talks about not wanting to update the hardware clock, but does this actually achieve that goal?

I've read the [timedated](https://www.freedesktop.org/wiki/Software/systemd/timedated/) spec and some of its source, and it seems that it will only update the clock when explicitly asked to, or when one calls `SetLocalRTC` with the `fix_system` parameter set to `false`. The spec seems to suggest calling `SetLocalRTC(local_rtc=true, fix_system=true, interactive=false)` in our situation:

> Use fix_system=true in installers and livecds where the RTC is probably more reliable than the system time.

though maybe we can leave well alone and hope it'll be okay. Did you look into this?